recipe/meta.yaml

```diff
@@ -24,13 +25,16 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python >=3.6.2
-    - astroid >=2.9.0,<2.10
+    - python
+    - astroid >=2.11.6,<2.14
```

Comment:
Looking in [the upstream setup.cfg](https://github.com/PyCQA/pylint/blob/v2.14.5/setup.cfg#L50), the astroid pinning should be `>=2.11.6,<2.12.0`.